src/screens/LanguageSelect/LanguageButton.js

```diff
@@ -1,13 +1,17 @@
 import React from "react";
 import { Button } from "react-native-elements";
 import PropTypes from "prop-types";
+
+// eslint-disable-next-line import/no-unresolved
+import { FIRST_INTRO_SCREEN } from "constants/introductionScreens";
+
 import { styles } from "./styles";
 
 const LanguageButton = props => {
   const { title, langKey, navigation } = props;
   /* eslint-disable-next-line */
   const onPress = key => {
-    navigation.navigate("auth");
+    navigation.navigate(FIRST_INTRO_SCREEN);
```

Comment:
With this, user will have to choose language and view intro screens every time he closes the app and opens it again. I think it's better to not change "auth" route for now, since we still haven't figured out how to show screens on first opening only ([task for it](https://jira.epam.com/jira/browse/EPMCUXDA-6913))

Reply:
According to designs and Denis Kargin clarification back button on first introduction screen should navigate to language selection also introduction screens should be displayed once like language screen by first app open.